src/System.Net.Security/src/System/Net/Security/SslStreamInternal.cs

```diff
@@ -63,7 +55,7 @@ private void FreeReadBuffer()
         {
             if (_internalBufferFromPinnableCache)
```

Comment:
Now that it's no longer the "pinnable cache", let's rename this to something else, maybe just "_internalBufferFromCache".  Same goes for related locals elsewhere, like "wasPinnable".